mm2src/coins/z_coin.rs

```diff
@@ -474,6 +477,12 @@ impl ZCoin {
                 .await?
                 .tx_result?;
 
+        // Store any change outputs we created in this transaction by decrypting them with our keys
+        // and saving them to the wallet database for future spends
+        store_change_output(self.consensus_params_ref(), &self.z_fields.light_wallet_db, &tx)
+            .await
+            .error_log();
```

Comment:
Why log the error and not return it?

Reply:
tbh honest, since the functionality was originally omitted from our fork of `librustzcash`, I figured it might not be a major issue if the function fails, so I opted to log an error instead. I also wanted a review on it